test/unit/logger/logger.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('Fail Queue Tests', () => {
```

Comment:
Wrong parking.

Reply:
its not, the describe message is wrong. 
so the test is there just to setup the test folder. 

I will change it tho.